rust/operator-binary/src/controller.rs

```diff
@@ -1025,7 +1025,7 @@ fn container_ports(trino: &TrinoCluster) -> Vec<ContainerPort> {
 }
 
 fn readiness_probe(trino: &TrinoCluster) -> Probe {
-    let port_name = if trino.https_port_enabled() {
+    let port_name = if trino.get_client_tls().is_some() {
```

Comment:
How come these don't use `expose_https_port` now that they are equivalent?

Reply:
done